CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+ - Support for Ruby 2.7 [[#29]]
+ - Gem metadata [[#29]]
+
+### Removed
+ - Support for Ruby 2.3 [[#29]]
+ - Support for Ruby 2.4 [[#29]]
+
+[29]: https://github.com/envato/zxcvbn-ruby/pull/29
+
```

Comment:
I think GitHub automatically adds a link to PRs whenever you add a `#` before its number.

```suggestion
 - Support for Ruby 2.7 [#29]
 - Gem metadata [#29]

### Removed
 - Support for Ruby 2.3 [#29]
 - Support for Ruby 2.4 [#29]
```

Reply:
GitHub does convert `#number` to PR/Issue links, but this Changelog may be viewed elsewhere, so I've followed @orien's lead from https://github.com/envato/unwrappr/pull/66 of being explicit.